deploy/olm-catalog/service-assurance-operator/0.1.0/service-assurance-operator.v0.1.0.clusterserviceversion.yaml

```diff
@@ -8,12 +8,15 @@ metadata:
           "apiVersion": "infra.watch/v1alpha1",
           "kind": "ServiceAssurance",
           "metadata": {
-            "name": "saf-default"
+            "name": "cloud1"
           },
           "spec": {
-            "eventsEnabled": false,
-            "metricsEnabled": true,
-            "haEnabled": false,
+            "events": {
+              "enabled": false
```

Comment:
You can't make this a map in Ansible Operators (well technically you can, but let me explain why it's a bad idea).

When you used mapped values with Ansible Operators instead of keeping everything flat (single plane), then if you ever add anything to the `metrics` or `events` mapping, then you need to re-define the entire map if you want to change just one value. The entire map is overridden, so let let's say you have `metrics.enabled` and `metrics.highAvail` and you just want to override one of those values, then you _must_ define both. If you don't then it'll just be a non-existent parameter when the Ansible runs.

The easiest way to avoid this confusion is to define everything flat like we've been doing.

Reply:
Thanks for injecting this thinking early. Asking the user to keep the entire config in sync with whatever is in the default map is truly unfair and causes lots of trouble. I've suffered through this in Chef, Puppet, and Helm, and the solution (staying flat) is SO SIMPLE, you just need to stick to it.